CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.4+1
```

Comment:
This should be 2.0.5. See [this document](https://go/versioning-dart-repo-packages) for details on our versioning scheme.